client/src/features/visualizer-threejs/VisualizerInstance.tsx

```diff
@@ -167,11 +171,22 @@ const VisualizerInstance: React.FC<RouteComponentProps<VisualizerRouteProps>> =
         };
     }, [runListeners]);
 
+    useEffect(() => {
```

Comment:
Why was this added ? There an identical `useEffect` just above this `useEffect`

Reply:
Merge debris :(